src/ophys_etl/modules/suite2p_registration/__main__.py

```diff
@@ -141,6 +168,17 @@ def run(self):
             # our custom reference image creation code, this dataset will
             # be empty.
             f.create_dataset('ref_image', data=suite2p_args['refImg'])
+            # Write a copy of the configuration output of this dataset into the
+            # HDF5 file.
+            args_copy = copy.deepcopy(self.args)
+            suite_args_copy = copy.deepcopy(suite2p_args)
```

Comment:
Doesn't creating `args_copy` effectively do this for you?

Reply:
I'm mostly making doubly sure that the args that we store in the HDF5 file are the values we modified.